### PR TITLE
Fixes #74 - Integrate prettier into eslint

### LINKED ISSUE #74: Enforce prettier or eslint style via pre-commit hook (or something else)?
There's like 50 ways to deal with this, but it's probably a good time to do it now so we don't keep reviewing style nits.

https://prettier.io/ is one popular option, it has eslint integration, nice VSCode / editor hooks etc.

Or perhaps a more strict eslint config with a command to run `--fix`? There's a few popular choices out there.

Personally I don't really care, as long as we can code, use some command to fix things up, then commit.

### PR DESCRIPTION
**Pull request by miketaylr**
OK, this PR adds prettier to eslint in addition to a pre-commit hook. So no need to worry about configuring your editor to work with prettier (but you can of course) -- just write JS and commit it.

### CODE REVIEW

**Comment by miketaylr** [comment]
We should land #127 and #124 first, and then rebase this PR against those.

**Comment by miketaylr** [comment]
(just pushed the rebased 124 stuff)

**Comment by miketaylr** [comment]
A useful hint for those that use vscode: to enable auto-prettier on save, make sure you have the eslint addon installed and put `"eslint.autoFixOnSave": true` in vscode settings.json

(I had installed the prettier addon, which worked manually, but never on save...)

File: package.json [hunk 1]
```diff
@@ -14,27 +14,44 @@
...
     "@octokit/plugin-retry": "^2.2.0",
     "@octokit/plugin-throttling": "^2.6.0",
     "@octokit/rest": "^16.28.2",
+    "add": "^2.0.6",
     "escape-string-regexp": "^2.0.0",
+    "eslint-config-prettier": "^6.7.0",
+    "eslint-plugin-prettier": "^3.1.1",
     "googleapis": "^40.0.0",
     "node-fetch": "^2.5.0",
+    "prettier": "^1.19.1",
```
Comment (past) [inline package.json]: Aren't these all devDependencies?

Comment (miketaylr) [reply package.json]: Yeah, should be. I guess I didn't pass the `--dev` flag when I installed here. Will fix.

**Pull request merged**